src/include/gurt/atomic.h

```diff
@@ -19,6 +20,10 @@
 #define ATOMIC _Atomic
 #endif
 
+#else
+#define ATOMIC
```

Comment:
(style)  storage class should be at the beginning of the declaration

Reply:
I'd mark this as negative.